pkg/cimfs/mount_cim.go

```diff
@@ -63,3 +64,36 @@ func Unmount(volumePath string) error {
 
 	return nil
 }
+
+// MountMergedCims mounts the given merged CIM (usually created with `CreateMergedCim`) at the a volume with
+// given GUID. The `cimPaths` MUST be identical to the `cimPaths` passed to `CreateMergedCim` when creating
+// this merged CIM.
+func MountMergedCims(cimPaths []string, mergedCimPath string, mountFlags uint32, volumeGUID guid.GUID) (string, error) {
```

Comment:
Are any flags currently supported?

Reply:
All of the standard mount flags that are already defined [here](https://github.com/microsoft/hcsshim/blob/main/internal/hcs/schema2/cim_mount.go) are supported. I will add two more flags to this for blocked CIMs when I open that PR.